File: api/src/main/java/com/aa/weatherprediction/WeatherPredictionApplication.java

package com.aa.weatherprediction;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.SpringApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@SpringBootApplication
@EnableDiscoveryClient
public class WeatherPredictionApplication {

    public static void main(String[] args) {
        SpringApplication.run(WeatherPredictionApplication.class, args);
    }
}

File: weather-prediction-api/src/main/kotlin/com/aa/weatherprediction/WeatherPredictionApplication.kt
package com.aa.weatherprediction

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@EnableDiscoveryClient
class WeatherPredictionApplication

fun main(args: Array<String>) {
	runApplication<WeatherPredictionApplication>(*args)
}
